lib/ravelin.rb

```diff
@@ -1,5 +1,39 @@
-require "ravelin/version"
+require 'date'
```

Comment:
Can you explain the reasons why you think it is a good idea / have to to import everything here?